keysign/discover.py

```diff
@@ -42,17 +42,19 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
-            msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
-            key_data, success, message = msg_tuple
-            if key_data:
+            if BluetoothReceive is not None:
+                self.bt = BluetoothReceive(self.bt_port)
+                msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
+                key_data, success, message = msg_tuple
+                if key_data:
                 # If we found the key
-                returnValue((key_data, success, message))
-
-        key_data = None
-        success = False
-        message = ""
-        returnValue((key_data, success, message))
+                    returnValue((key_data, success, message))
+            else:
+                log.info("PyBluez is probably missing.")
+    key_data = None
```

Comment:
You messed up the indentation. They need 4 more spaces otherwise they are out of the `start` function.